microcosm_flask/audit.py

```diff
@@ -95,6 +146,11 @@ def parse_response(response):
         return response, 200
 
 
+@defaults(
```

Comment:
The factory now takes defaults and constructs its own copy of the decorator that passes these in as options.